mvnw

```diff
@@ -8,7 +8,7 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
-#    https://www.apache.org/licenses/LICENSE-2.0
+#    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
this is the result of `mvn wrapper:wrapper` code generation
so i  guess it's better to keep it as is and probably to fix it first in upstream project

Reply:
Cool I am happy with that. I will do some digging on the issue with the failing test on windows, as it only happens on this branch. (5/5 times) didnt happen when I ran  it on main. I'll let you know if it can be fixed or what the RCA is